src/Symfony/Component/Runtime/Tests/FrankenPhpWorkerRunnerTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime\Tests;
+
+require_once __DIR__.'/frankenphp-function-mock.php';
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\TerminableInterface;
+use Symfony\Component\Runtime\Runner\FrankenPhpWorkerRunner;
+
+interface TestAppInterface extends HttpKernelInterface, TerminableInterface
+{
+}
+
+/**
+ * @author Kévin Dunglas <kevin@dunglas.fr>
```

Comment:
this annotation doesn't mean anything on test cases, except fame maybe :)
if you really want to give credit, then, this should be on git blame IMHO (not sure Kevin is going to miss this file in his legacy anyway :) )